### PR TITLE
suite-nonsplit wishlist endpoint

### PR DESCRIPTION
**Pull request by himanshu-pal-metacube**
## PR Description
CC-16851

### CODE REVIEW

File: tests/api/suite_nonsplit/glue/wishlist_endpoints/wishlists/positive.robot [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+     And Response body parameter should be:    [data][id]    ${wishlist_id} 
+     And Response body parameter should not be EMPTY:    [data][relationships]
+     And Response body parameter should not be EMPTY:    [data][relationships][wishlist-items][data][0]
+     And Response include element has self link:    wishlist-items
+     And Response include element has self link:    concrete-products
+     And Response include element has self link:    product-labels
+     [Teardown]    Run Keywords    I send a DELETE request:    /wishlists/${wishlist_id}
+    ...  AND    Response status code should be:    204
+
+Retrieves_wishlist_with_items_in_concreate
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/wishlist_endpoints/wishlists/positive.robot:160]: please rename the test - it sounds strange.
Retrieves_wishlist_with_items_including_concrete_products

File: tests/api/suite_nonsplit/glue/wishlist_endpoints/wishlists/positive.robot [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+     And Response body parameter should be:    [data][type]    wishlists
+     And Response body parameter should be:    [data][attributes][name]    ${random}
+     And Response body parameter should be:     [data][attributes][numberOfItems]    1
+     And Response body parameter should be:    [data][id]    ${wishlist_id}
+     And Response body parameter should not be EMPTY:    [data][relationships]
+     And Response body parameter should not be EMPTY:    [data][relationships][wishlist-items][data][0]
+     AND Response body parameter should not be EMPTY:    [data][attributes][createdAt]
+     AND Response body parameter should not be EMPTY:    [data][attributes][updatedAt]
+     And Response include element has self link:    wishlist-items
+     And Response include element has self link:    concrete-products
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/wishlist_endpoints/wishlists/positive.robot:181]: we should check includes more, not just that they have selflinks.
Eg
And Response should contain the array larger than a certain size: 
And Response include should contain certain entity type:

File: tests/api/suite_nonsplit/glue/wishlist_endpoints/wishlists/positive.robot [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+     AND Response body parameter should be:    [data][attributes][name]    ${random}
+     AND Response body parameter should be greater than:    [data][attributes][numberOfItems]    -1
+     AND Response body parameter should not be EMPTY:    [data][id]
+     AND Response body parameter should not be EMPTY:    [data][attributes][createdAt]
+     AND Response body parameter should not be EMPTY:    [data][attributes][updatedAt]
+     AND Response body parameter should not be EMPTY:    [data][links][self]
+     [Teardown]    Run Keywords    I send a DELETE request:    /wishlists/${wishlist_id}
+    ...  AND    Response status code should be:    204
+
+# #Get_Request
```
[the remark this answers is not in the document]
Comment (himanshu-pal-metacube) [reply tests/api/suite_nonsplit/glue/wishlist_endpoints/wishlists/positive.robot]: Fixed

File: tests/api/suite_nonsplit/glue/wishlist_endpoints/wishlists/positive.robot (hunk 2 above)
Comment (himanshu-pal-metacube) [reply tests/api/suite_nonsplit/glue/wishlist_endpoints/wishlists/positive.robot:180]: Fixed

**Comment by himanshu-pal-metacube** [comment]
resolved conflict

**Review by Katerynaa-Lavryk — APPROVED** [review]

**Pull request merged**